gazelle/WORKSPACE

```diff
@@ -38,6 +38,10 @@ load("@rules_python//python:repositories.bzl", "py_repositories")
 
 py_repositories()
 
+load("//:internal_dev_deps.bzl", "internal_dev_deps")
+
+internal_dev_deps()
```

Comment:
I'm not familiar with the old `WORKSPACE` stuff. Is there a way to mark this as a dev-only dep like is done with bzlmod?

Reply:
WORKSPACE files are, essentially, all dev-only. (nothing in them gets "inherited" and used by consumers, as happens in MODULE files).